mkdocs.yml

```diff
@@ -52,7 +52,7 @@ pages:
   - ProteinPaint: 'Data_Portal/Users_Guide/proteinpaint_lollipop.md'
   - Sequence Reads: 'Data_Portal/Users_Guide/proteinpaint_bam.md'
   - OncoMatrix: 'Data_Portal/Users_Guide/oncomatrix.md'
-  - Cohort MAF: 'Data_Portal/Users_Guide/cohortMAF.md'
+  # - Cohort MAF: 'Data_Portal/Users_Guide/cohortMAF.md'
```

Comment:
@wwysoc2 please check that this is right/ok.

Reply:
No idea if commenting out the line will work, let's merge it and see how it builds.